src/rules/de.toml

```diff
@@ -7,12 +7,13 @@ quote_start_with_letter = true
 needs_punctuation_end = false
 needs_letter_start = true
 needs_uppercase_start = true
-allowed_symbols_regex = "[\u0020-\u007BäöüßÄÖÜ„“‚‘’–\\.]"
+allowed_symbols_regex = "[\u0020-\u007BäöüßÄÖÜ„“‚‘’–\\.?!]"
```

Comment:
Ah, I see why " is allowed, it's part of this unicode range. Then we probably indeed should add it to the even count rule. Additionally ! and ? are already part of this range, so it doesn't need to be added. However that range also includes some characters that we probably not want, such as $, % and others. https://unicode-table.com/en/#0021

Sorry, should have had a closer look before...

Reply:
Basically. What I would suggest here is the following:

```
allowed_symbols_regex = "[\u0020A-za-zäöüßÄÖÜ\"„“‚‘’–\\.?!()]"
```